client/src/Components/Login/LoginFrom.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLocation, useNavigate, Link } from 'react-router-dom';
+import {
+  Form, Input, Button, message,
+} from 'antd';
+import axios from 'axios';
+import { LoginFormType } from '../../utils';
+
+interface loginProps {
+  values: LoginFormType,
+}
+
+function LoginForm() {
+  const [loading, setLoading] = useState(false);
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const location: any = useLocation();
+  const onFinish = async (values: loginProps) => {
+    try {
+      setLoading(true);
+      setTimeout(() => {
+        setLoading(false);
+      }, 1000);
+      const loginResult = await axios.post('/api/v1/login', values);
+      const newLink = `/user/${loginResult.data.data.providerName}`;
+      message.success(t('success-login'));
+      console.log(location);
```

Comment:
remove these two lines,
Actually, we want to redirect user  after success so no need to display a message for user